### PR TITLE
Migrate prettier from prettier/prettier to prettier/pre-commit

### PR DESCRIPTION
**Pull request by joao-p-marques**
See https://github.com/prettier/prettier/issues/9459

### CODE REVIEW

**Comment by joao-p-marques** [comment]
Problem with Still depending on https://github.com/prettier/pre-commit/issues/16 fixed

**Comment by joao-p-marques** [comment]
When https://github.com/prettier/pre-commit/pull/17 and https://github.com/prettier/pre-commit/pull/18 are merged, we can remove the extra dependencies and args.

File: .vscode/settings.json [hunk 1]
```diff
@@ -32,7 +32,6 @@
   "python.formatting.provider": "black",
   "python.linting.flake8Enabled": true,
   "python.linting.pylintEnabled": true,
-  "python.pythonPath": "${workspaceRoot}/.venv/bin/python",
```
Comment (Yajo) [inline .vscode/settings.json:35]: I guess you should leave this here.

**Review by Yajo — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (joao-p-marques) [reply .vscode/settings.json:35]: Sorry, VSCode keeps making this change on it's own and I committed it by accident.

Comment (joao-p-marques) [reply .vscode/settings.json]: Changes made

**Review by Yajo — APPROVED** [review]

**Pull request merged**